internal/proxy/middleware/authz/authz.go

```diff
@@ -219,7 +219,7 @@ func (c *Authz) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 		permissionAttributes[key] = value
 	}
 
-	isAuthorized := false
+	isAuthorized := true
```

Comment:
we should not change the default to `true`
we should keep this false so the behaviour is "normally denied"

the permission itself needs to have a default behaviour.

Reply:
can you share how are you visualising the end to end flow for this case?

Reply:
the use case is - we want to check certain permissions when the sink type is bq or gcs.
if there is any other sink, till now we were not evalutaing any permission (by not configuring the auth middleware in rules). 
now by keeping isAuthorized to true, we are allowing all the traffic for which we dont want permission check.
for those its needed, we will have permissions setup which will be selected based on expression.

Reply:
@AkarshSatija views?